lib/tentacat/hooks.ex

```diff
@@ -3,7 +3,8 @@ defmodule Tentacat.Hooks do
   alias Tentacat.Client
 
   @moduledoc """
-  The Repository Webhooks API allows repository admins to manage the post-receive hooks for a repository.
+  The Repository Webhooks API allows repository admins to manage the
+  post-receive hooks for a repository.
```

Comment:
do we need this change? 🤔

Reply:
Follow the overall file line wrapping length.